x-pack/plugins/ml/server/lib/alerts/jobs_health_service.ts

```diff
@@ -99,17 +131,36 @@ export function jobsHealthServiceProvider(
             const jobState =
               jobsStats.find((jobStats) => jobStats.job_id === jobId)?.state ?? 'failed';
             return {
-              ...datafeedStats,
+              datafeed_id: datafeedStats.datafeed_id,
+              datafeed_state: datafeedStats.state,
               job_id: jobId,
               job_state: jobState,
             };
           })
           .filter((datafeedStat) => {
             // Find opened jobs with not started datafeeds
-            return datafeedStat.job_state === 'opened' && datafeedStat.state !== 'started';
+            return datafeedStat.job_state === 'opened' && datafeedStat.datafeed_state !== 'started';
           });
       }
     },
+    /**
+     * Gets jobs that reached soft or hard model memory limits.
+     * @param jobIds
+     */
+    async getMmlReport(jobIds: string[]): Promise<MmlTestResponse[]> {
```

Comment:
Do we want this to check only for `opened` jobs? Currently you also get an alert for `closed` jobs that have hit the hard or soft limit.

Reply:
Added check in 17f75f4edb8158a68cc912c323f10e0727eb52af